cmd/skopeo/sync.go

```diff
@@ -771,6 +793,7 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) (retErr error) {
 					continue
 				}
 			}
+
```

Comment:
(Non-blocking: I think keeping the counter increment immediately after the action being counted is a bit cleaner than adding the blank line here.)